src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -792,44 +517,5 @@ private void OnLoggingSettingsGui()
             }
             EditorGUI.EndDisabledGroup();
         }
-
-        private void ResetInitDirectory()
-        {
-            initDirectory = Utility.UnityProjectPath;
-            GUIUtility.keyboardControl = GUIUtility.hotControl = 0;
-        }
-
-        private void ForceUnlockFile(object obj)
```

Comment:
We should make sure we track this in an issue for fixing, we'll need to do this as part of the initial file locking support.

Reply:
Who are you?